util/bloom_test.cc

```diff
@@ -399,21 +401,22 @@ TEST_P(FullBloomTest, FilterSize) {
   bool some_computed_less_than_denoted = false;
   // Note: enforced minimum is 1 bit per key (1000 millibits), and enforced
```

Comment:
Seem like we also need to fix this comment as we now drop "the minimum is 1 bit per key (1000 millibits)"